docs/abtest.rst

```diff
@@ -54,9 +54,9 @@ different.
 Create the variant page like you would a new page. Make sure it is ``noindex``
 and does not have a ``canonical`` url.
 
-```
-{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
-```
+.. code-block:: python
```

Comment:
should be `.. code-block:: jinja` and have a blank line before the content.

Reply:
Significant white space is my nemesis.